MovieGallery/Pages/OurTeam.cshtml

```diff
@@ -10,10 +10,10 @@
         <h1 class="display-4 font-weight-bold jumbotron">Meet Our Team</h1>
     </div>
     <div class="row no-gutters">
-        <div class="col-sm-5">
-            <img src="\images\Rutuja.png" class="card-img-top h-75" alt="Rutuja Sawant">
-        </div>
         <div class="col-sm-7">
+            <img src="\images\Rutuja.png" class="card-img-top h-75" alt="Image of Rutuja Sawant who is a front end developer" />
+        </div>
+        <div class="col-sm-5">
```

Comment:
made the alt tag more descriptive, so that if the image does not load, the user is able to see the description and understand what the image is all about.